### ISSUE TITLE
Imports with a dash in the name throws an error. 

### ISSUE THREAD

**Issue by askho** [r-murphy/babel-plugin-transform-modules-ui5#4]
When attempting to import a module for side effects only the transormation will throw an error. 
IE: Doing this type of import:
```javascript
import "MyWorkOrdersVeg/vendor/pouchdb/pouchdb";
import  "MyWorkOrdersVeg/vendor/pouchdb/pouchdb-find";
import "MyWorkOrdersVeg/vendor/pouchdb/pouchdb-upsert";
```

will throw this error: 

```
SyntaxError: unknown: Unexpected token, expected , (1:169)
> 1 | sap.ui.define(["bchui5/util/BCHLogger", "bchui5/util/common", "pouchdb", "pouchdb-upsert", "pouchdb-find"], function (BCHLogger, bchui5_util_common, __pouchdb, __pouchdb-upsert, __pouchdb-find) {
    |                                                                                                                                                                          ^
  2 |     const guid = bchui5_util_common["guid"];
  3 | 
  4 |     const PouchDB = window.PouchDB;
    at Parser.pp$5.raise (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp.expect (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:1749:33)
    at Parser.pp$2.parseBindingList (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3101:12)
    at Parser.pp$1.parseFunctionParams (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:2395:22)
    at Parser.pp$1.parseFunction (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:2385:8)
    at Parser.pp$3.parseFunctionExpression (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3760:17)
    at Parser.pp$3.parseExprAtom (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3722:19)
    at Parser.pp$3.parseExprSubscripts (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3344:19)
    at Parser.pp$3.parseExprListItem (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:4312:16)
    at Parser.pp$3.parseCallExpressionArguments (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3573:20)
    at Parser.pp$3.parseSubscripts (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3533:31)
    at Parser.pp$3.parseExprSubscripts (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3504:15)
    at Parser.pp$3.parseMaybeUnary (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3344:19)
    at Parser.pp$3.parseExpression (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:3306:19)
    at Parser.pp$1.parseStatement (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:1906:19)
    at Parser.pp$1.parseBlockBody (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:2268:21)
    at Parser.pp$1.parseTopLevel (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:1778:8)
    at Parser.parse (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:1673:17)
    at parse (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babylon/lib/index.js:7305:37)
    at File.parse (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babel-core/lib/transformation/file/index.js:517:15)
    at File.parseCode (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babel-core/lib/transformation/file/index.js:602:20)
    at /Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babel-core/lib/transformation/pipeline.js:49:12
    at File.wrap (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babel-core/lib/transformation/file/index.js:564:16)
    at Pipeline.transform (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/node_modules/babel-core/lib/transformation/pipeline.js:47:17)
    at Object.transpileFileAsync (/Users/leonho/Documents/BCH/bch-dev-tools/dt-transpiler/lib/handlers/js.js:104:17)
    at <anonymous>:null:null

```

However if I hack it by exporting it to a random variable it will work:

```javascript 
import * as _ignore1 from "MyWorkOrdersVeg/vendor/pouchdb/pouchdb";
import * as _ignore2 from "MyWorkOrdersVeg/vendor/pouchdb/pouchdb-find";
import * as _ignore3 from "MyWorkOrdersVeg/vendor/pouchdb/pouchdb-upsert";
import BCHLogger from "bchui5/util/BCHLogger";
import { guid } from "bchui5/util/common";
import * as _ignore4 from "pouchdb";
import * as _ignore5 from "pouchdb-upsert";
import * as _ignore6 from "pouchdb-find";
console.log(_ignore1, _ignore2, _ignore3);
const PouchDB = (window as any).PouchDB as PouchDB.Static;
```

**Comment by askho** [comment]
Did some more testing. It looks like it has nothing to do with side effects only imports, but because the name has a dash in yet.

**Comment by r-murphy** [comment]
Thanks for reporting. Fixed in version 6.17.2 by converting dashes to spaces in the variable names.
